test/unit/alphabet/semi_alphabet_constexpr_test_template.hpp

```diff
@@ -0,0 +1,168 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include <seqan3/alphabet/concept.hpp>
+
+#if SEQAN3_WITH_CEREAL
```

Comment:
I don't see cereal tested anywhere?
Also, this can be replaced with `#include <seqan3/test/cereal.hpp>`

Reply:
good point... I cannot see it either. I copied this over from the alphabet_test. I'll check if I can remove them